packages/namekit-react/src/components/Tooltip.tsx

```diff
@@ -25,7 +25,7 @@ export function Tooltip({
   const handleClose = () => setOpen(false);
 
```

Comment:
Something has broken with the tooltips on the NameGuard website. Please ensure everything is working.

![CleanShot 2024-07-01 at 03 32 17](https://github.com/namehash/nameguard/assets/126201998/024f1ad6-d88e-4d06-9148-5e25bdcc048e)

Reply:
@lightwalker-eth this is related to Tailwind prefixes appliance, I will review this with @notrab asap!

Reply:
We should have to import the styles into the NameGuard website for the package that is now exporting the component for it to continue working as it were

Reply:
Resolved, thank you @notrab !